app/components/windows/EditStreamInfo.vue.ts

```diff
@@ -107,6 +125,10 @@ export default class EditStreamInfo extends Vue {
       // If the stream info pre-fetch failed, we should try again now
       this.refreshStreamInfo();
     }
+
+    if (this.isTwitch) {
+      this.allTwitchTags = await this.twitchService.getAllTags();
```

Comment:
there's no need to inject the twitch service directly here, we can use the `getPlatformService` command to use it

Reply:
We need to typecast if we do it that way. Is this what we want?